assets/javascripts/discourse/models/chat-channel.js

```diff
@@ -46,4 +46,20 @@ export default RestModel.extend({
 
     return !READONLY_STATUSES.includes(this.status);
   },
+
+  isDirectMessageChannel() {
+    return this.chatable_type === CHATABLE_TYPES.directMessageChannel;
+  },
+
+  isTopicChannel() {
+    return this.chatable_type === CHATABLE_TYPES.topicChannel;
+  },
+
+  isCategoryChannel() {
+    return this.chatable_type === CHATABLE_TYPES.categoryChannel;
+  },
+
+  isTagChannel() {
+    return this.chatable_type === CHATABLE_TYPES.tagChannel;
+  },
```

Comment:
All these would be better as getters (e.g. `get isDirectMessageChannel() {}`) so they could be used like regular properties (`channel.isDirectMessageChannel`)

Reply:
There is no chef's kiss reaction on github, so :chefs_kiss: . Always forget about getters, I haven't used them enough!